install/kubernetes/Makefile

```diff
@@ -16,12 +17,21 @@ DEV_VERSION_REGEX := '[0-9]\+\.[0-9]\+\.[0-9]\+-dev'
 CILIUM_CHART_REGEX := '\([vV]ersion:\) '$(VERSION_REGEX)
 CILIUM_TAG_REGEX := '\(tag:\) \(v'$(VERSION_REGEX)'\|latest\)'
 CILIUM_PULLPOLICY_REGEX := '\(pullPolicy:\) .*'
+ULTIMATE_OPTIONS := \
+    --set global.hubble.enabled=true \
+    --set global.hubble.listenAddress=":4244" \
```

Comment:
Should we enable this by default without any warning to the user? (I know there's a warning in the logs but that would be it)

Reply:
@aanm Are you saying that we should enable these options by default for all cilium installs or asking whether we should warn users when these settings are enabled?

Reply:
> @aanm Are you saying that we should enable these options by default for all cilium installs or asking whether we should warn users when these settings are enabled?

warn users when these settings are enabled since this might be unnoticed.

Reply:
@aanm @Rolinh i was thinking of simply putting a warning in the cilium doc whenever we mention `kubectl apply -f experimental-install.yaml`. i'd be happy to add warnings in other place if we already have some sort of convention to warn users about experimental features with security implications.

Reply:
@aanm How would you suggest the warning should be delivered to users? In cilium status? IMO if it is not possible for the user to use the experimental options without explicitly invoking `experimental-install.yaml` on the command line warnings in the docs that document such use should be enough.

Reply:
Note that we can generate custom text in the Helm NOTES output depending on specific helm options.

Reply:
this is how the warning is worded right now: https://github.com/cilium/cilium/pull/11882/files#diff-0806f700bf4c3d27724e99235d5f13a8R47

Reply:
@joestringer yeah i think it makes sense to add a warning in NOTES.txt if `global.hubble.listenAddress` is set. i will open a separate PR for that since it's not directly related to experimental-install.yaml.